config/puma.rb

```diff
@@ -10,7 +10,7 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch("PORT") { 3000 }
+port        ENV.fetch('PORT')
```

Comment:
This change is unnecessary

Reply:
why ?

Reply:
cors looks for ENV['PORT'] anyway, 3000 is the default value